### DISCUSSION TITLE
Styles missing on Forge Server

### DISCUSSION THREAD

**Discussion by davidwhicker** [spatie/laravel-pdf#13] [General]
I have run into an issue whereby the production version does not bring in any of the styles on the blade view. The local version on my Mac is fine. Code is identical. 

Only difference is running the commands to install on a forge server from your on the requirments section.

**Comment by freekmurze** [comment]
Since it's working locally, the problem is likely to be caused by versions not being the same on production.

**Reply by davidwhicker** [reply]
Thanks. It's got me stumped. It doesnt seem to be pulling in any of the styles found in app.css. On production deployment it runs npm run build and works fine everywhere expect the PDF itself. I'll keep thinking
